frontend/src/components/Log.test.js

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { act } from 'react-dom/test-utils';
 import Log from './Log';
+import moment from 'moment-timezone';
 
 let container;
 
 beforeEach(() => {
   container = document.createElement('div');
   document.body.appendChild(container);
+  moment.tz.setDefault('UTC');
```

Comment:
@gulyaasjaanos though this is a cool solution, on frontend we have to be careful about adding libraries just for the sake of few usages and testing. moment is pretty huge (react is 5k, moment is 455k).
so I'd go with the `cross-env TZ=UTC react-scripts test` if it works on windows (I've tried it and it works on a mac)
because `cross-env` can be a devDependency only

Reply:
unfortunately cross-env seems to fail on Win10, just like the global-settings approach
I've found a workaround however:
since the actual logentry reader will be much more complex than the mocked one, I moved it from the componenet to a service -> that way it proved to be mockable; I set the date-time there in the timestamps, taking into account the timezone